crates/nu-std/std/core/mod.nu

```diff
@@ -6,8 +6,8 @@ let dt = (datetime-diff (date now) 2019-05-10T09:59:12-07:00)
 
 let banner_msg = $"(ansi green)     __  ,(ansi reset)
 (ansi green) .--\(\)°'.' (ansi reset)Welcome to (ansi green)Nushell(ansi reset),
-(ansi green)'|, . ,'   (ansi reset)based on the (ansi green)nu(ansi reset) language,
```

Comment:
not sure how i feel about removing this line. it was kind of a big point to sophia, to state that nushell is a language too, not just a shell.

Reply:
Not sure then - Any other ideas on where to put it?

Reply:
Or other ideas on how to reword it?  I know the line has some history, but *"based on the nu language"* is a bit ... odd.  First, we don't refer to the language separately from the shell really.  I totally agree that we want to get the point across that it is a shell *and* language, but it's not *"based on"* - They are one in the same.  And we generally don't refer to (lower-case) `nu` as being the language and Nushell as being the shell.

Reply:
There were plans for https://github.com/nu-lang. We may return to that at some point. In one sense, it's honoring the nushell creators since one of their main goals was to create a programming language.

I'd just put a line above ellie or below her with the version.

Reply:
I think in the existing wording there may be a minor risk of confusion with the Objective-C lisp `nu` if you google for "nu language". So I like the new wording

Reply:
I like the original text better.
```
     __  ,
 .--()°'.' Welcome to Nushell, based on the
'|, . ,'   nu language, where all data is
 !_-(_\    structured! v0.100.1 (windows-x86_64)
```
or
```
     __  , Welcome to Nushell, based
 .--()°'.' on the nu language, where
'|, . ,'   all data is structured!
 !_-(_\    v0.100.1 (windows-x86_64)
```